multy_transaction/internal/amount.h

```diff
@@ -0,0 +1,73 @@
+#ifndef MULTY_TRANSACTION_INTERNAL_AMOUNT_H
+#define MULTY_TRANSACTION_INTERNAL_AMOUNT_H
+
+#include "third-party/mini-gmp/mini-gmp.h"
+
+#include <string>
+
+#include <stdint.h>
+
+struct Amount
+{
+    explicit Amount(const char* value);
+    explicit Amount(int32_t value = 0);
+    Amount(const Amount& other);
+
+    ~Amount();
+
+    void set_value(const char* value);
+    std::string get_value() const;
```

Comment:
add comment that return string with unnecessary symbols '\0'

Reply:
Fixed method